engines/director/lingo/xlibs/movutils.cpp

```diff
@@ -104,9 +105,14 @@ III    mGetMessage           --Get mouse/key messages from the application messa
 
 namespace Director {
 
-const char *const MovUtilsXObj::xlibName = "MovUtils";
+const char *const MovUtilsXObj::xlibNames[] = {
+	"MovUtils",
+	"MovieUtilities"
```

Comment:
```suggestion
	"MovieUtilities",
	nullptr
```

Otherwise, we have dead loops in those `exposeXObject()` calls

Reply:
done